### PR TITLE
Log bot "version" on error handling

### PR DESCRIPTION
**Pull request by rnevet**
When logging an exception also log the number of commits on current branch, will allow to easily identify if reporter is on master and updated.

## TESTING STAGE
in progress

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [n/a] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [n/a] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [ ] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

File: modules/Configuration.py [hunk 1]
```diff
@@ -241,3 +241,15 @@ def get_plugins_config():
     if config.has_option("BOT", "plugins"):
         active_plugins = map(str.strip, config.get("BOT", "plugins").split(','))
     return active_plugins
+
+
+# gets the git commit count as version for master.
+def get_bot_version():
```
Comment (Evanito) [inline modules/Configuration.py]: This function belongs in the Data module, it has nothing to do with the Config.

**Comment by laxdog** [comment]
Good idea, we definitely need this. 

Was just thinking there, we could add a second place to look up the version number if it's not a git repo. Then we can at least manually update that every once in a while, it could be the actual release version.

**Comment by rnevet** [comment]
@Evanito If the git command doesn't return a number a default version number '3.0.0' will be returned. We can update this number from time to time, so we have some indication of the bot version. 

I currently don't have time to include the CLI command, can I merge without?

File: modules/Data.py [hunk 2]
```diff
@@ -141,3 +141,14 @@ def truncate(f, n):
...
+
+
+# gets the git commit count as version for master.
+def get_bot_version():
+    import subprocess
+    output = subprocess.check_output(["git", "rev-list","--count", "HEAD"])
+    try:
+        int(output)
+        return output
+    except ValueError:
```
Comment (laxdog) [inline modules/Data.py]: You need to catch 'Exception' here I think. Or work out the other exceptions. 

If it's not a git repo, output won't exist and it'll be a NameError, if git doesn't exist it'll be an OSError. Not sure what else there could be.

Comment (rnevet) [reply modules/Data.py]: Fixed.

**Pull request merged**